custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -1147,6 +1153,23 @@ def _get_requirements_lines(reqs_file_path):
 
         return ret_docker_image
 
+    def _generate_runtime_report_file(self, fit_mem_usage: float, pred_mem_usage: float) -> None:
+        """
+        Saves information related to running a fit pipeline.  All data is reported in Mb
+
+        Parameters:
+            fit_mem_usage: Memory footprint of running the fit job
+            pred_mem_usage: Memory footprint of running the check for prediction side effects
+        """
+        report_information = {
+            "fit_memory_usage": fit_mem_usage,
+            "prediction_memory_usage": pred_mem_usage,
+            "input_filesize": os.path.getsize(self.options.input) / 1e6,
```

Comment:
I think this is getting the size of the directory, not the contents in the directory. Needs to be recursive